dcos_launch/sample_configs/aws-onprem-enterprise-with-helper.yaml

```diff
@@ -0,0 +1,24 @@
+---
+launch_config_version: 1
+deployment_name: dcos-onprem-with-new-stack
+installer_url: https://downloads.mesosphere.com/dcos-enterprise/testing/master/dcos_generate_config.ee.sh
+platform: aws
+provider: onprem
+aws_region: us-west-2
+os_name: cent-os-7-dcos-prereqs
+instance_type: m4.xlarge
+bootstrap_instance_type: m4.xlarge
+dcos_config:
+  cluster_name: My Awesome DC/OS
+  resolvers:
+    - 8.8.4.4
+    - 8.8.8.8
+  dns_search: mesos
+  master_discovery: static
+  exhibitor_storage_backend: static
+num_masters: 1
+num_private_agents: 1
+num_public_agents: 1
+ssh_user: centos
+key_helper: true
+dcos_version: 1.12
```

Comment:
When did we start introducing dcos_version in the config?

Reply:
This seems like a new field to me, because we specify dcos_version by the URL itself and never had to deal with this.

Reply:
@orsenthil when enable_selinux field was introduced, we needed a way to check dcos version to only enable selinux on version >=1.12
We decided to do this with DC/OS detect version repo to find the DC/OS download URI, because the version is not in the URI for DC/OS PRs. 
Example: https://s3.amazonaws.com/downloads.mesosphere.io/dcos-enterprise/testing/pull/1296/commit/29eaee2b33552d146ddf9970f374f7626e056420/dcos_generate_config.ee.sh

Reply:
dcos_version is injected in the config by DC/OS version detect build step in TeamCity

Reply:
Got it. a link to that PR would help too. Thank you!

Reply:
I mean, the dependency between "when enable_selinux ", and this `dcos_version` field and the information that is populated from the teamcity. We could discuss this offhand for better understanding.